honeycombio/resource_marker_settings.go

```diff
@@ -0,0 +1,105 @@
+package honeycombio
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	honeycombio "github.com/honeycombio/terraform-provider-honeycombio/client"
+)
+
+func newMarkerSetting() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMarkerSettingCreate,
+		ReadContext:   resourceMarkerSettingRead,
+		UpdateContext: nil,
```

Comment:
You wrote the function for this below but didn't tell the resource to use it
```suggestion
		UpdateContext: resourceMarkerSettingUpdate,
```